mautic/mautic
# fix default value when creating new date or time company custom field

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for features or enhancements / 3.2 for bug fixes 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #9512  




#### Description:
Fix the bug that displays when you add a default value while creating a new company custom (date or time) field
#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create new company time field
3. add a default value
4. Save and see that it works
